HLTriggerOffline/JetMET/python/Validation/SingleJetValidation_cfi.py

```diff
@@ -17,7 +17,8 @@
 
 from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
 phase2_common.toModify(SingleJetMetPaths,
-                       PatternJetTrg = "HLT_(AK4)?PFPuppiJet([0-9])+(_v[0-9]+)?$",
+                       PatternJetTrg  = "HLT_(AK4)?PFPuppiJet([0-9])+(_v[0-9]+)?$",
+                       PatternMetTrg  = "HLT_PFPuppiMET+[TypeOne140_PFPuppiMHT140]+(_v[0-9]+)?$",
```

Comment:
@missirol @SohamBhattacharya Could you please advise on string for MET path? Or ping responsible person.

I think I will stop after this PR as we should be ready to handle Phase-2 HLT DQM. And let Phase-2 HLT developers to handles all paths.

Thanks.

Reply:
I think the following can be a reasonable starting point
```python
^HLT_(AK4|AK8)?PF(Puppi)?Jet[0-9]+(_v[0-9]+)?$
^HLT_PF(Puppi)?MET(TypeOne)?[0-9]+(_PF(Puppi)?MHT[0-9]+)?(_v[0-9]+)?$
```
but I'll let @SohamBhattacharya @beaucero comment. (similarly for the Muon case)

Reply:
I agree about the Jet/MET paths. For the muons I would suggest starting with the following, in case the isolation WP and the pt thresholds change in the future.
`HLT_Mu[0-9]+_TrkIso[A-Z]+_Mu[0-9]+_TrkIso[A-Z]+_DZ_FromL1TkMuon(_v[0-9]*)?$`
`HLT_Mu[0-9]+_Mu[0-9]+_FromL1TkMuon(_v[0-9]*)?$`
`HLT_Mu[0-9]+_FromL1TkMuon(_v[0-9]*)?$`